kafka-producer/src/main/java/producer/KafkaProducerApplication.java

```diff
@@ -0,0 +1,12 @@
+package producer;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class KafkaProducerApplication {
```

Comment:
Hm, doesn't look like it produces anything

Reply:
@ktereshchenko auto generated

Reply:
@ktereshchenko Do I have to put "@EnableKafka" there or ?

Reply:
I'm just thinking out loud - it shouldn't be named `Producer` if it doesn't produce anything. The `@EnableKafka` annotation enables `@KafkaListener`, so if you use it, then yes. But you have `KafkaProducerConfig` class, that is actually more appropriate place to add the annotation if you need it. Here is the good article on the topic - https://www.baeldung.com/spring-kafka